filters/apiusagemonitoring/spec.go

```diff
@@ -17,15 +17,15 @@ const (
 	unknownPlaceholder = "{unknown}"
 	noMatchPlaceholder = "{no-match}"
 	noTagPlaceholder   = "{no-tag}"
-
-	regexUrlPathPart     = `.+`
-	regexOptionalSlashes = `\/*`
 )
 
 var (
-	log                         = logrus.WithField("filter", Name)
-	regexVarPathPartCurlyBraces = regexp.MustCompile("^{([^:{}]+)}$")
-	regexVarPathPartColon       = regexp.MustCompile("^:([^:{}]+)$")
+	log                   = logrus.WithField("filter", Name)
+	regexpMultiSlash      = regexp.MustCompile("/+")
+	regexpLeadingSlash    = regexp.MustCompile("^/?")
+	regexpTrailingSlash   = regexp.MustCompile("/?$")
+	rexexpSlashColumnVar  = regexp.MustCompile("/:([^:{}/]*)")
+	rexexpCurlyBracketVar = regexp.MustCompile("{([^{}]*)}")
```

Comment:
Should we exclude `/` (slashes) from variable names? Is it likely or possible, that pattern matcher libraries implement it simila?